timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResults.vue

```diff
@@ -236,32 +238,90 @@ export default {
+              id: analysis.timeline.id,
+              name: analysis.timeline.name,
+              color: analysis.timeline.color,
+              last_analysissession_id: analysis.analysissession_id,
+              analysis_status: multiResults.analysis_status,
+              results: multiResults.results,
+            }
+          } else {
+            // this is not a multi analyzer, so add the results for this timeline
+            perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name] = {
+              id: analysis.timeline.id,
+              name: analysis.timeline.name,
+              color: analysis.timeline.color,
+              last_analysissession_id: analysis.analysissession_id,
+              analysis_status: analysis.status[0].status,
+              results: [
+                {
+                  verdict: analysis.result,
+                  created_at: analysis.created_at,
+                  analysis_status: analysis.status[0].status,
+                },
+              ]
+            }
           }
         }
 
         if (
           perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].last_analysissession_id <
           analysis.analysissession_id
         ) {
-          // this timeline is already in the results for this analyzer but check if the session is newer and update it
-          perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].created_at = analysis.created_at
-          perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].verdict = analysis.result
-          perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].last_analysissession_id =
-            analysis.analysissession_id
-          perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].analysis_status =
-            analysis.status[0].status
+          // there is a newer session for this analyzer on this timeline. Update the results.
+          if (perAnalyzer[analysis.analyzer_name].analyzerInfo.is_multi) {
+            multiResults = this.getAnalyzerMultiResults(analyses, analysis.analyzer_name, analysis.analysissession_id, analysis.timeline.id)
+            perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].last_analysissession_id =
+              analysis.analysissession_id
+            perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].results[0].analysis_status = multiResults.analysis_status
+            perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].results = multiResults.results
+          } else {
+            perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].results[0].created_at = analysis.created_at
+            perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].results[0].verdict = analysis.result
+            perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].last_analysissession_id =
```

Comment:
The `last_analysissession_id` seems to be the same in both branches of the `if-else`. So you could pull it out:

```javascript
perAnalyzer[analysis.analyzer_name].timelines[analysis.timeline.name].last_analysissession_id =
              analysis.analysissession_id
if (perAnalyzer[analysis.analyzer_name].analyzerInfo.is_multi) {
  // ...
} else {
  // ...
}
```